sdks/python/apache_beam/pipeline_test.py

```diff
@@ -439,6 +440,21 @@ def test_dir(self):
              if not attr.startswith('_')]))
 
 
+class RunnerApiTest(unittest.TestCase):
+
+  def test_simple(self):
+    """Tests serializing, deserializing, and running a simple pipeline.
+
+    More extensive tests are done at pipeline.run for each suitable test.
```

Comment:
Yes, pipeline.run has been modified to do a round trip through protos for all possible pipelines.